examples/ansible/setup_firewalld.yml

```diff
@@ -0,0 +1,76 @@
+---
+# Ansible Playbook: Setup firewalld using Podman on SUSE ALP Dolomite
+# Description: This Ansible playbook automates the deployment of the firewalld using Podman on SUSE ALP Dolomite.
+# The deployment adds firewall capability to ALP Dolomite to define the trust level of network connections or interfaces.
+# Key Considerations:
+# - The container image utilizes the system's dbus instance. Thus, dbus and polkit configuration files must be installed initially.
+# - The systemd service and its configuration file allow the container to start and stop via systemd with Podman as the container manager.
+# - The `/usr/local/bin/firewall-cmd` serves as a wrapper to invoke firewall-cmd inside the container, with both Docker and Podman being supported.
+# Based on: "Running firewalld using Podman on SUSE ALP Dolomite". Documentation available at:
+# [https://documentation.suse.com/alp/dolomite/single-html/firewalld-podman-alp-dolomite/]
+
+- name: Setup firewalld using Podman on SUSE ALP Dolomite
+  hosts: alphost
+  become: true
+  vars:
+    workload:
+      name: firewalld
+      image: registry.opensuse.org/suse/alp/workloads/tumbleweed_images/suse/alp/workloads/firewalld
+
+  tasks:
+    - name: Ensure firewalld is not installed in the host system
+      ansible.builtin.package:
+        name: "{{ workload.name }}"
+        state: absent
+      register: remove_firewalld_result
+
+    - name: Handle reboot, if needed
+      when: remove_firewalld_result is changed
+      block:
+        - name: Print message requesting immediate reboot
+          ansible.builtin.debug:
+            msg: >-
+               The 'firewalld' package has been uninstalled via transactional-update to ensure compatibility with the intended containerized workload.
+
+        - name: Request immediate reboot  # noqa no-handler
+          ansible.builtin.fail:
+            msg: "Reboot required"
+
```

Comment:
```suggestion
- name: Check if firewalld is installed locally
  stat:
    path: "rpm -q firewalld"
  register: local_firewalld
- name: Fail if firewalld is installed locally
  ansible.builtin.fail:
    msg: "Firewalld is installed locally. Please remove it before installing this container."
  when: local_firewalld.rc != 0
```

Reply:
stat module only work with path ie. /usr/sbin/firewall, using ansible.builtin.package_facts.

~~~
"msg": "The conditional check 'local_firewalld.rc != 0' failed. The error was: error while evaluating conditional (local_firewalld.rc != 0): 'dict object' has no attribute 'rc'. 'dict object' has no attribute 'rc'\
~~~